cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -4673,7 +4761,19 @@ macro(build_awssdk)
       aws-cpp-sdk-core
       aws-c-event-stream
       aws-checksums
-      aws-c-common)
+      aws-c-common
+      aws-c-auth
+      aws-c-cal
+      aws-c-compression
+      aws-c-http
+      aws-c-io
+      aws-c-mqtt
+      aws-c-s3
+      aws-c-sdkutils
+      s2n
+      crypto
+      ssl
```

Comment:
Are they artifacts by `s2n-tls`?
Then we should not create `AWS::crypto` and `AWS::ssl`. We should create `AWS::s2n-tls` and associate `libcrypto.a` and `libssl.a` with `AWS::s2n-tls`.

Reply:
It turns out they are not needed. I've removed these two.